src/components/c-tabs.vue

```diff
@@ -7,6 +7,7 @@
       <li v-for="tab in tabs"
           :key="tab.id"
           :class="b('tab-item')"
+          role="tabitem"
```

Comment:
Shouldn't this be role tab?

```suggestion
          role="tab"
```